packages/cli/src/commands/init.ts

```diff
@@ -1,11 +1,48 @@
+import * as open from 'open';
+import { Answers } from 'inquirer';
 import { prompt } from '../client';
 import { promptIntroArray } from './init.consts';
+import { HttpServer } from '../server';
+import { SERVER_PORT, SERVER_HOST, REDIRECT_ROUTH, API_OAUTH_URL } from '../constants';
+
+let answers: Answers;
```

Comment:
I think it's better to try and avoid global variables and instead use functions with clear input and output. This can help with better predictable functions with input and output.

Reply:
Good point I will try to avoid global variables in the future! 👍